docs/TroubleShooting.md

```diff
@@ -6,6 +6,38 @@
 
 ## Common
 ### Issues on Kconfig-frontend
+If you are using gperf3.1 and kconfig-frontends-4.11.0.1,
```

Comment:
We had better split two issues by sub-title.
```
#### build break on hconf

#### execution error on mconf
```

Reply:
Fixed it